usaspending_api/api_docs/api_documentation/download/list_agencies.md

```diff
@@ -0,0 +1,75 @@
+## [List Agencies](#download-status)
+**Route:** `/api/v2/bulk_download/list_agencies/`
+
+**Method:** `POST`
+
+This route lists all the agencies and the subagencies or federal accounts associated under specific agencies.
+
+### Request example
+
+```
+{
+    "agency": 1,
```

Comment:
Remove comma to prevent JSON parse error. 
`agency` 1 gives "Agency ID not found", so maybe also update to use an existing agency id.

Reply:
Dan is looking into it and making a comment on this pr

Reply:
Good call, using `"agency": 14` works